tutorials/SingleQubitGates/SingleQubitGates.ipynb

```diff
@@ -404,6 +404,13 @@
     "}"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Exectuting the cell below with the `%trace` command will execute the `PauliGates_Demo` function defined above, and will print all the gates nicely at the botom of the output have a look."
```

Comment:
```suggestion
    "In the previous tutorials we used `%simulate` command to run the Q# code on the full-state simulator. Here we will use `%trace` command: it will run the code on the full-state simulator as well, but will also print the circuit diagram of the run after the output."
```